amethyst_renderer/src/formats/texture.rs

```diff
@@ -163,7 +163,7 @@ impl PngFormat {
 }
 
 impl SimpleFormat<Texture> for PngFormat {
-    const NAME: &'static str = "JPEG";
+    const NAME: &'static str = "PNG";
```

Comment:
Nice catch :)

Reply:
I only caught it because the string was concatenated into a very confusing error message :P

Reply:
Which confusing error message? Is my error handling code broken?

Reply:
@torkleyy no no, it's fine, it just told me it couldn't load a JPEG when I was trying to load a PNG, because this string was concatenated into the message.  I appreciate your vigilance, but if your asset loading produced confusing error messages for me I would have complained :)